src/Whoops/Handler/JsonResponseHandler.php

```diff
@@ -84,7 +84,12 @@ public function handle()
             header('Content-Type: application/json');
         }
 
-        echo json_encode($response);
+        if (version_compare(phpversion(), '5.4', '<')) {
```

Comment:
Infact, it would be MUCH better to just check if `JSON_PARTIAL_OUTPUT_ON_ERROR` is defined.